setup.md

```diff
@@ -70,30 +70,33 @@ If you run into any difficulties, please request help before the workshop begins
         press enter to prepend Anaconda to your `PATH` (this makes the Anaconda
         distribution the default Python).
 
-> ## conda version
->
-> If you already have anaconda installed at your computer, make sure you have an up-to-date version of conda running.
-> See [these instructions](https://docs.anaconda.com/anaconda/install/update-version/) for updating conda.
->
-{: .callout}
-
 ## Installing the required packages
 
-Open a terminal and type the command (note that installing tensorflow causes keras to
-be installed too):
+[Conda](https://docs.conda.io/projects/conda/en/latest/) is the package management system associated with Anaconda and runs on Windows, macOS and Linux. 
+If you have installed Anaconda successfully, Conda should be already available in your system and work regardless of the operating system.
+Make sure you have an up-to-date version of Conda running.
+See [these instructions](https://docs.anaconda.com/anaconda/install/update-version/) for updating Conda. 
+{: .callout}
+
+Open a terminal and type the command:
 ~~~
-$ conda install tensorflow seaborn scikit-learn pandas
+conda install tensorflow seaborn scikit-learn pandas
 ~~~
-{: .language-bash}
+{: .source}  
+Note that installing Tensorflow causes Keras to be installed too.
```

Comment:
This phrase does not reflect reality and mixes terms, think keras as an API rather than a library.
```suggestion
Note that by installing a modern version of Tensorflow, you have access to the `keras` module in it.
```

Reply:
I would prefer a more compact formulation like:  
> Note that modern versions of Tensorflow make Keras available as a module.

Reply:
Fair enough, sounds like a plan. Please use this or directly commit it by virtue of:

```suggestion
Note that modern versions of Tensorflow make Keras available as a module.
```